dotCMS/src/main/java/com/dotcms/rest/api/v1/contentImport/ContentImportHelper.java

```diff
@@ -0,0 +1,157 @@
+package com.dotcms.rest.api.v1.contentImport;
+
+import com.dotcms.jobs.business.api.JobProcessorScanner;
+import com.dotcms.jobs.business.api.JobQueueManagerAPI;
+import com.dotcms.jobs.business.processor.JobProcessor;
+import com.dotcms.jobs.business.processor.Queue;
+import com.dotcms.rest.api.v1.JobQueueManagerHelper;
+import com.dotcms.rest.api.v1.temp.DotTempFile;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.web.WebAPILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.util.Logger;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.liferay.portal.model.User;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.servlet.http.HttpServletRequest;
+import java.lang.reflect.Constructor;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@ApplicationScoped
+public class ContentImportHelper {
+
+    private static final String CMD_PREVIEW = "preview";
+    private static final String CMD_PUBLISH = "publish";
+
+    private JobQueueManagerAPI jobQueueManagerAPI;
```

Comment:
set to final

Reply:
I think you can Valentino, but in the empty CDI constructor you set the value to null, I have done it and should not cause problems.